Modules/Segmentation/SuperPixel/test/itkSLICImageFilterGTest.cxx

```diff
@@ -23,7 +23,7 @@
 
 #include "itkCommand.h"
 
-#include "itkTestDriverIncludeRequiredIOFactories.h"
+#include "itkTestDriverIncludeRequiredFactories.h"
```

Comment:
Will this change be required in external code, such as remote modules? If so, we could keep a file called `itkTestDriverIncludeRequiredIOFactories.h` with content `#include "itkTestDriverIncludeRequiredFactories.h"`.

Reply:
All tests relying on the ITKTestKernel architecture already include `itkTestDriverIncludeRequiredIOFactories.h`, so this line will only appear in older tests not relying on the test kernel. Looking through ITK I'm only able to find this line in GTest and "v3" segmentation/registration tests that are built as separate executables. Are we still supporting these in ITK 5, or are remote modules expected to use the test kernel?

Reply:
"v3" registration is still fully supported.

Reply:
Re-added `itkTestDriverIncludeRequiredIOFactories.h` for backwards compatibility